cloudinary_cli/modules/sync.py

```diff
@@ -160,6 +146,36 @@ def push(self):
         if upload_errors:
             raise Exception("Sync did not finish successfully")
 
+    def pull(self):
+        """
+        Pulls changes from the Cloudinary folder to the local folder.
+        """
+        download_results = {}
+        download_errors = {}
+        if not self._handle_unique_local_files():
+            return False
+
+        files_to_pull = self.unique_remote_file_names | self.out_of_sync_remote_file_names
+
+        if not files_to_pull:
+            return True
+
+        logger.info(f"Downloading {len(files_to_pull)} files from Cloudinary")
+        downloads = []
+        for file in files_to_pull:
+            remote_file = self.remote_files[file]
+            local_path = path.abspath(path.join(self.local_dir, file))
```

Comment:
why are we naming this local_path when we're using the abspath method?

Reply:
@strausr, here `local` means that it is on the current file system and not `remote` on cloudinary.